nodejs/week2/homework/meal-sharing/src/backend/app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
```

Comment:
As developer is import to pay attention to the unit tests and make sure they pass
```
Test Suites: 1 failed, 1 total
Tests:       10 failed, 8 passed, 18 total
Snapshots:   0 total
Time:        1.395 s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
```